coretex/_task/_local.py

```diff
@@ -34,6 +34,9 @@
 from ..networking import networkManager
 
 
+EXPERIMENT_CONGIF_PATH = "./experiment.config"
```

Comment:
Let's make this an instance of `pathlib.Path` class like so: `Path(".", "experiment.config")`